charmcraft/commands/init.py

```diff
@@ -71,12 +71,12 @@
     ├── README.md              - Frontpage for your charmhub.io/charm/
     ├── requirements.txt       - PyPI dependencies for your charm, with `ops`
     ├── src
-    │   └── charm.py           - Minimal operator using Python operator framework
+    │   └── charm.py           - Minimal operator using Python operator framework
```

Comment:
Tabs to spaces?  The output still looks OK on the terminal?

Reply:
Non-breaking spaces to spaces, and still looks fine in the terminal.